src/components/Participant/Participant.js

```diff
@@ -74,13 +75,16 @@ function Participant({
   return (
     <div
       className={classNames(
-        'relative group p-1 border-2 bg-white',
+        'participant relative group bg-white',
         'transition duration-150 ease-in-out',
         focus || 'border-white',
         focus && 'border-secondary shadow-md',
         speaking && 'border-green-300'
-      )}>
-      <div className="relative bg-gray-100">
+      )}
+      style={{width: "calc(var(--partSlotWidth) - 6px)", margin: "3px", padding: "4px", "border-width": "2px"}}>
```

Comment:
Question: Could we place these width and height attributes with `calc` directly in the css file?

Reply:
Yes, we can place them in a `.participant` CSS class which we can define in the _components layer_ at https://github.com/jangouts/jangouts/blob/5ab3baa06a64295461f14c5ed8bc8a1dd018796a/src/styles/index.css#L24-L48

 I guess @ancorgs didn't go for it because we are (almost) not defining CSS classes by ourselves and using Tailwind instead. BTW, I'd like to change that in the future. Wanna you join?

Reply:
> we can place them in a `.participant` CSS class 

If so, please add documentation there too. At least a reference to the `.js` file where its meaning and the reason for its existence is explained.